hack/jenkins/verify-jenkins-jobs.sh

```diff
@@ -38,6 +38,13 @@ kubernetes-pull-build-test-e2e-gce
 kubernetes-pull-test-unit-integration
 kubernetes-update-jenkins-jobs
 kubernetes-verify-jenkins-jobs
+kubernetes-upgrade-1.0-current-release-gce
+kubernetes-upgrade-gce-1.1-master
+kubernetes-upgrade-gce-stable-current-release
+kubernetes-upgrade-gke-1.0-current-release
+kubernetes-upgrade-gke-1.0-master
+kubernetes-upgrade-gke-1.1-master
+kubernetes-upgrade-gke-stable-current-release
```

Comment:
The comment above is now out of date; please add a note about multi-jobs too.

This is going to suck, because every time we add a new multi-job we'll have to add it here.  :-1: